src/vs/workbench/contrib/files/browser/fileActions.contribution.ts

```diff
@@ -41,6 +42,17 @@ registry.registerWorkbenchAction(new SyncActionDescriptor(ShowOpenedFileInNewWin
 registry.registerWorkbenchAction(new SyncActionDescriptor(CompareWithClipboardAction, CompareWithClipboardAction.ID, CompareWithClipboardAction.LABEL, { primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KEY_K, KeyCode.KEY_C) }), 'File: Compare Active File with Clipboard', category.value);
 registry.registerWorkbenchAction(new SyncActionDescriptor(ToggleAutoSaveAction, ToggleAutoSaveAction.ID, ToggleAutoSaveAction.LABEL), 'File: Toggle Auto Save', category.value);
 
+const fileCategory = nls.localize('file', "File");
```

Comment:
@Tyriar I think moving these commands to `fileActions.contribution.ts` is good, however I am not 100% convinced, the "Open Local..." commands can live in `browser` namespace. I think we can only ever offer them in `electron-browser` at least until browsers give us API to open random files locally. 

Related to that, we should not show an "Open Local" button in the custom file/folder picker unless we run natively.

//cc @alexr00

Reply:
@aeschli the "Show Local" is only provided if we have more than 1 availableFileSystem provided to the picker. It looks like for the web only the single availableFileSystem should be passed to the picker in the browser.

Reply:
@alexr00 shouldn't the heuristic rather be that "Show Local" only appears if there is a registered file system provider for the `file://` scheme?

Reply:
Right now we assume that if there is more than one available file system, then one of the available file systems must be `file://`. If that will not be the case then I will update it to actually check and make sure that one of the schemes is file.

Reply:
@alexr00 yes please do this update because in the web we will possibly have multiple file system providers that are all remote and never local